usecases/evaluate_scenario/evaluate_scenario.go

```diff
@@ -161,12 +161,14 @@ func EvalScenario(
 	// Compute outcome from score
 	var outcome models.Outcome
 
-	if score < publishedVersion.Body.ScoreReviewThreshold {
-		outcome = models.Approve
-	} else if score < publishedVersion.Body.ScoreRejectThreshold {
+	if score >= publishedVersion.Body.ScoreRejectThreshold {
```

Comment:
reversing the logic here, more in line with how the thresholds are named (though it's of course equivalent)